index.html

```diff
@@ -17,7 +17,11 @@ <h1>Holiday Road</h1>
     <div class="content--left">
 
       <!-- Container for our three <select> elements: dropdown--park, dropdown--oddity, dropdown--eatery -->
-      <div class="dropdownContainer"></div>
+      <div class="dropdownContainer">
+        <section class="dropdown dropdownParkContainer"></section>
```

Comment:
Can we call the shared class here `dropdownContainer` instead of just `dropdown`? All of the `<select>` elements within these `<section>` also have `dropdown` class and stuff might get weird if both the `<select>` and its container have the same class name like that.